src/ConnectionFactory.php

```diff
@@ -119,7 +119,7 @@ public function createConnection(array $params, ?Configuration $config = null, ?
             $connection = DriverManager::getConnection(...array_merge([$params, $config], $eventManager ? [$eventManager] : []));
             $params     = $this->addDatabaseSuffix(array_merge($connection->getParams(), $overriddenOptions));
             $driver     = $connection->getDriver();
-            /** @psalm-suppress InvalidScalarArgument Bogus error, StaticServerVersionProvider implements Doctrine\DBAL\ServerVersionProvider  */
+            /** @phpstan-ignore arguments.count (DBAL 2.x doesn't accept an argument) */
```

Comment:
```suggestion
            /** @phpstan-ignore arguments.count (DBAL 3.x doesn't accept an argument) */
```

I'm on a phone, so please double-check. But I think we've made that change going from 3 to 4.